ontology/hyperm.html

```diff
@@ -88,18 +88,18 @@ <h2>Introduction</h2>
 <section class="specterm"><h4><a href="#hyperm-hastarget">hyperm:hasTarget</a></h4><p>IRI: https://www.w3.org/2019/wot/hypermedia#hasTarget</p><p><strong>a OWL Class</strong></p><em>hasTarget</em> - <span>target IRI of a link or submission target of a form.</span><table><tbody></tbody></table></section>
 <section class="specterm"><h4><a href="#hyperm-returns">hyperm:returns</a></h4><p>IRI: https://www.w3.org/2019/wot/hypermedia#returns</p><p><strong>a OWL Class</strong></p><em>returns</em> - <span>This optional term can be used if, e.g., the output communication metadata differ from input metdata (e.g., output contentType differ from the
      input contentType). The response name contains metadata that is only valid for the reponse messages.</span><table><tbody></tbody></table></section>
-<section class="specterm"><h4><a href="#hyperm-hasanchor">hyperm:hasAnchor</a></h4><p>IRI: https://www.w3.org/2019/wot/hypermedia#hasAnchor</p><p><strong>a OWL Class</strong></p><em>hasAnchor</em> - <span>By default, the context, or anchor, of a link conveyed in the Link header field is the URL of the representation it is associated with, as defined in RFC7231, Section 3.1.4.1, and is serialized as a URI.</span><table><tbody></tbody></table></section></section></section><section><h2>Term Mapping</h2><table><thead><tr><th>JSON String</th><th>Ontology Term</th></tr></thead><tbody><tr><td>anchor</td><td><a href="#hyperm-hasanchor">hyperm:hasAnchor</a></td></tr>
-<tr><td>type</td><td><a href="#hyperm-hintsatmediatype">hyperm:hintsAtMediaType</a></td></tr>
-<tr><td>Form</td><td><a href="#hyperm-form">hyperm:Form</a></td></tr>
+<section class="specterm"><h4><a href="#hyperm-hasanchor">hyperm:hasAnchor</a></h4><p>IRI: https://www.w3.org/2019/wot/hypermedia#hasAnchor</p><p><strong>a OWL Class</strong></p><em>hasAnchor</em> - <span>By default, the context, or anchor, of a link conveyed in the Link header field is the URL of the representation it is associated with, as defined in RFC7231, Section 3.1.4.1, and is serialized as a URI.</span><table><tbody></tbody></table></section></section></section><section><h2>Term Mapping</h2><table><thead><tr><th>JSON String</th><th>Ontology Term</th></tr></thead><tbody><tr><td>op</td><td><a href="#hyperm-foroperationtype">hyperm:forOperationType</a></td></tr>
```

Comment:
This is just "render noise": as far as I can tell, the definitions in this file are generated in a random order each time... but the same definitions are there as before.  Probably this file should not be checked in...